src/iuse_actor.cpp

```diff
@@ -867,7 +867,43 @@ long pick_lock_actor::use( player &p, item &it, bool, const tripoint& ) const
     return it.type->charges_to_use();
 }
 
+iuse_actor *deploy_furn_actor::clone() const {
+    return new deploy_furn_actor( *this );
+}
+
+void deploy_furn_actor::load( JsonObject &obj )
+{
+    furn_type = static_cast<furn_id>( obj.get_string( "furn_type" ) );
```

Comment:
You can only use `int_id`s after all objects have been loaded from JSON. See `set_furn_ids`, which sets up some global `int_id`s and is called in the finalization step after all furniture objects have been loaded. The `int_id` can *not* be determined when the object (the furniture type in this case) has not yet been loaded. Use a `string_id` instead.

And also: why are you casting? A constructor is enough.